tests/plugins/divarea/manual/detached.md

```diff
@@ -0,0 +1,63 @@
+@bender-tags: 4.17.0, bug, 4462
+@bender-ui: collapsed
+@bender-ckeditor-plugins: divarea,toolbar,basicstyles,link,format,sourcearea,elementspath,undo
+
+**Note:** Open dev console to track eventual errors.
+
+1. Reattach editor with double click on "Toggle" button.
+
+	**Expected:** Editor shows with the same data.
+
+	**Unexpected:** Editor data is lost.
+
+2. Click inside the content editing area.
+
+	**Expected:** Content is editable. Elementspath (bottom bar) updates.
+
+	**Unexpected:** Content is not editable. Elemetspath (bottom bar) remains empty.
+
+3. Write any text in editor.
+
+4. Reattach editor with double click on "Toggle" button.
+
+	**Expected:** Editor shows with the same data.
+
+	**Unexpected:** Editor data is lost.
+
+5. Click inside the content editing area.
+
+	**Expected:** Content is editable. Elementspath (bottom bar) updates.
+
+	**Unexpected:** Content is not editable. Elemetspath (bottom bar) remains empty.
```

Comment:
I'm not sure about this one. Generally speaking - if the content is not editable - verifying `elementspath` is pointless.

With this PR fix, `elementspath` indeed remains the same after reattach. There was a time during development - that this simply doesn't show anything even if the content was visible and editable. So, if after reattach & focus inside the editor it remains the same - that indicates that the plugin is working. Also, this scenario is quite long already - so maybe we could stay as it is?

Reply:
Ok, in general I don't have a problem with checking elementspath at all - I think we just need to change the word `updates` as it may suggest that something should change. So instead let's just say that it shows correct path or sth like this.